install/mysql-install.sh

```diff
@@ -24,9 +24,32 @@ $STD apt-get install -y \
   mc
 msg_ok "Installed Dependencies"
 
+PS3="Please select a Major Release: "
```

Comment:
Why not echo/whiptail for choosing?

Reply:
Good question.. I will look into it and submit an updated PR.